### ISSUE TITLE
Modules config file

### ISSUE THREAD

**Issue by JoseEspinosa** [cbcrg/nf-benchmark#8]
Modules included in nf-benchmark need may need their own config and this should be handle. A possible solution is to create a config for each module (`pipeline`, `benchmark`) and inside define a container for each of the processes if needed or a single one if this is the case.

**Comment by JoseEspinosa** [comment]
Another solution will be that config files of modules were also included when the module is included, could this make sense @pditommaso ?

**Comment by pditommaso** [comment]
Yeah, the latter is something it would be nice to have. Open an issue on NF repo.

**Comment by JoseEspinosa** [comment]
While this is not implemented in Nextflow, all the pipeline `configuration files` will be included by default.
To discriminate the configuration included this way from `nf-benchmark` configurations, the latter will be prefixed by `nfb-` see #26. Also related to #25.
